opensearch_py_ml/ml_models/sentencetransformermodel.py

```diff
@@ -657,6 +657,22 @@ def _add_apache_license_to_model_zip_file(self, model_zip_file_path: str):
         with ZipFile(str(model_zip_file_path), "a") as zipObj:
             zipObj.writestr("LICENSE", r.content)
 
+    def _add_third_party_copyrights_statements_to_model_zip_file(
+        self, third_party_copyrights_statements: str, model_zip_file_path: str
+    ):
+        """
+        Add Statements text for non Apache-2.0 licensed third party model. Add it to the model zip file at model_zip_file_path
+
+        :param third_party_copyrights_statements: Statements text for non Apache-2.0 licensed third party model. Should be put in the final artifact.
+        :type third_party_copyrights_statements: string
+        :param model_zip_file_path: Path to the model zip file
+        :type model_zip_file_path: string
+        :return: no return value expected
+        :rtype: None
+        """
+        with ZipFile(str(model_zip_file_path), "a") as zipObj:
+            zipObj.writestr("THIRD-PARTY", third_party_copyrights_statements)
```

Comment:
`THIRD-PARTY` can we put this as a static variable?

Reply:
Yes, ack